src/BreadcrumbErrorHandler.php

```diff
@@ -44,7 +44,7 @@ public function doHandleException($exception)
         }
 
         $this->client->leaveBreadcrumb(new Breadcrumb(
-            $this->client->translateSeverity($exception->getSeverity()),
+            Severity::fromString($this->client->translateSeverity($exception->getSeverity())),
```

Comment:
This seems a not useful double-run... Shouldn't we strip the severity translation from the client and put it in completely inside `Severity`?

Reply:
I completely agree but the client at the moment has a map (a part that we didn't refactor yet) that can be configured to customize which event severity gets assigned to an event according to the error level. Making this change here means refactoring that whole part and maybe a separate PR would better fit this purpose. It's something I want to do anyway soon or later